src/acp/ess2.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2022 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * :::{admonition} Experimental API
+ * :class: important
+ *
+ * The Access Control Policies proposal has not yet been reviewed for inclusion in the Solid spec.
+ * To enable early experimentation, solid-client exposes a low-level API. However, this API can and
+ * will include breaking changes in non-major releases. Additionally, for most applications, a
+ * higher-level API that is planned will be more applicable.
+ *
+ * Thus, the following export is *only* intended for experimentation by early adopters, and is not
```

Comment:
same comment about just a sentence about what acp_ess_2 is for, to support ESS v2.

Is it possible to have acp_ess_2 have the descriptions in its page rather than as references to the acp_ess_1?  https://solid-client-js-git-acp-ess-export-inrupt.vercel.app/modules/acp_ess_2.html#references

It makes it seem like acp_ess_1 is the more definitive one.  So, if only one can have the definition, I'd almost go for having it in acp_ess_2 and have acp_ess_1 reference the acp_ess_2 doc.

Reply:
Fixed as per previous comment (now includes an explicit statement about ESS 2.0.

Interestingly, it was enough to ammend the TypeDoc entrypoints ordering to get all functions descriptions in acp_ess_2 (and references in acp_ess_1): https://github.com/inrupt/solid-client-js/pull/1564/commits/143f52d90ee7ca2df9f4943ce57d660fa9f0de5c.

I'm not sure how to have them in both places I'm afraid, hope it's good enough.